### DISCUSSION TITLE
BLE sensors. When are they unavailable?

### DISCUSSION THREAD

**Discussion by host-felice** [arendst/Tasmota#19149] [Q&A]
Is there a way to detect when a ble sensor is no longer available? Like some kind of timeout? I mean, what if tasmota doesn’t see the sensor anymore for, say, 10-20-30 minutes?

Thank you for any hints

**Comment by barbudor** [comment]
Same problem, same answer: https://github.com/arendst/Tasmota/discussions/19112

**Comment by host-felice** [comment]
Thank you. I didn't find it while searching. I was investigating on ble only.
